account-integrations/safe/src/SafeECDSARecoveryPlugin.sol

```diff
@@ -22,35 +22,81 @@ interface ISafe {
     ) external returns (bool success);
 }
 
+interface ISafeECDSAPlugin {
+    function getOwner(address safe) external view returns (address);
+}
+
+struct ECDSARecoveryStorage {
+    address recoveryAccount;
```

Comment:
Consideration for the future: I'm starting to really like the term `Guardian` over `recoveryAccount`. I remember I really struggled to think of a good word for this with the BLS recovery.  It also seems like a lot of people in the ecosystem use Guardian as well.

Reply:
Ditto - I like that term too. Game to transition to it